tests/test_project/templates/admin/base_site.html

```diff
@@ -0,0 +1,16 @@
+{% extends 'admin/base_site.html' %}
+{% load static %}
+
+{% block extrastyle %}
+  {{ block.super }}
+  {% if OPENWISP_TEST_MODE %}
```

Comment:
Can we remove this variable too?

Reply:
We need this variable to disable transitions during tests. But now it is not being served by utils.

Reply:
Any specific reason to not add follow css in [menu-test.css](https://github.com/openwisp/openwisp-utils/blob/master/tests/test_project/static/menu-test.css) file?

Reply:
It will disable transition in the development environment too.